RibonDAO/app
# Feature/google pay

# Description

- Add google pay payment

# Necessary changes

- Put here any changes that must be done when this PR is merged
- eg: update an environment variable

## Does this pull request close any open issues?

- Put here the issue that must be closed with this PR
- eg: closes [#001]()

## Type of change

Please delete options that are not relevant.


## How to test

Put here all the things need to test this PR and verify your changes, also list any relevant details for tests (eg: have a wallet)

- Go to the contribution page
- Select the google pay option
- Confirm the payment in the google pay screen
Note: this must be done in a real device, since google pay does not work in simulators
